pkg/resources/marketplace/grpc_image_catalog_source_reconciler.go

```diff
@@ -0,0 +1,72 @@
+package marketplace
+
+import (
+	"context"
+	"fmt"
+
+	coreosv1alpha1 "github.com/operator-framework/operator-lifecycle-manager/pkg/api/apis/operators/v1alpha1"
+	"github.com/sirupsen/logrus"
+	k8serr "k8s.io/apimachinery/pkg/api/errors"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	k8sclient "sigs.k8s.io/controller-runtime/pkg/client"
+	"sigs.k8s.io/controller-runtime/pkg/reconcile"
+)
+
+type GRPCImageCatalogSourceReconciler struct {
+	Image     string
+	Context   context.Context
+	Client    k8sclient.Client
+	Namespace string
+	CSName    string
+}
+
```

Comment:
```suggestion

var _ CatalogSourceReconciler = &GRPCImageCatalogSourceReconciler{}

```

Blank statement to assert that `GRPCImageCatalogSourceReconciler` implements `CatalogSourceReconciler`

Reply:
:+1:  I'll add it

Reply:
Done